sky/clouds/utils/oci_utils.py

```diff
@@ -121,5 +123,13 @@ def get_profile(cls) -> str:
         return skypilot_config.get_nested(
             ('oci', 'default', 'oci_config_profile'), 'DEFAULT')
 
+    @classmethod
+    def get_default_image_os(cls) -> str:
+        # Get the default image OS. Instead of hardcoding, we give a choice to
+        # set the default image OS type in the sky's user-config file. (if not
+        # specified, use the hardcode one at last)
+        return skypilot_config.get_nested(('oci', 'default', 'image_os_type'),
+                                          'ubuntu')
```

Comment:
This is a shallow function, I would prefer to keep the content in `oci.py` alone.

Reply:
> This is a shallow function, I would prefer to keep the content in `oci.py` alone.

For this one, because we have other OCI related configurations like "compartment", "vcn_subnet", "image_tag_gpu", "oci_config_profile", etc in oci_utils.oci_config.  the default "image_os_type" is also an configuration item, would it be fine to set it there together with these OCI configurations?

Reply:
I see. This makes sense. I am ok with keeping it here.